src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1475,6 +1475,7 @@ table = try $ do
 
 inline :: MarkdownParser (F Inlines)
 inline = choice [ whitespace
+                , wikilink
```

Comment:
I like to keep related code grouped together, putting this next to `link` would be more logical.